README.md

```diff
@@ -183,12 +233,85 @@ collections:
 ```
 
 Notes:
-- The tmbdID can be either from a collection or an individual movie
-- You can specify more than one tmdb_id but it will pull the summary, poster, and background from only the first one.
-- Local posters/backgrounds are loaded over tmdb_poster/tmdb_background if they exist unless tmdb_poster/tmdb_background is also specified
+- The `tmdb_id` can be either from a collection or an individual movie
+- You can specify more than one `tmdb_id` but it will pull the summary, poster, and background from only the first one.
+- Local posters/backgrounds are loaded over `tmdb_poster`/`tmdb_background` if they exist unless `tmdb_poster`/`tmdb_background` is also specified
+
+#### TMDb List (List Type)
+
+###### Works with Movie and TV Show Libraries
+
+In addition to TMDb collections you can also build collections based off of TMDb Lists using `tmdb_list`.
+
+```yaml
+collections:
+  Top 50 Grossing Films of All Time (Worldwide):
+    tmdb_list: https://www.themoviedb.org/list/10
+```
+```yaml
+collections:
+  Top 50 Grossing Films of All Time (Worldwide):
+    tmdb_list: 10
+```
+
+#### TMDb Movie (List Type)
+
+###### Works with Movie Libraries
+
+You can also add individual movies to a collection using `tmdb_movie`.
+
+```yaml
+collections:
+  Anaconda:
+    tmdb_collection: https://www.themoviedb.org/collection/105995
+    tmdb_movie: https://www.themoviedb.org/movie/336560
```

Comment:
This is awesome. We should think about whether we want to do something similar for IMDb and Trakt

Reply:
im sure none of that would be very difficult

Reply:
let's get this pull going without any more features these can be in the next batch